packages/contracts/contracts/strategies/HackMoneyStrategy.sol

```diff
@@ -439,6 +446,7 @@ contract HackMoneyStrategy is HackMoneyStrategyBase, IHackMoneyStrategy {
      * @param collateralToLock collateral available to lock.
      * @return size how many options we can sell.
      */
+    // TODO: Never used. Remove?
```

Comment:
this one was used for _getRequiredCollateral in case we trade different type of options. For the current strategy not useful we can drop